src/OpenDB/test/unitTestsPython/helper.py

```diff
@@ -1,9 +1,9 @@
-import opendbpy as odb
+import opendb as odb
 import os
 
 
 def createSimpleDB():
-    db = odb.dbDatabase.create()
+    db = odb.create_db()
```

Comment:
Why the change?

Reply:
Agree this is likely to break users for not much benefit.

Reply:
create_db is a helper function that creates a database and link it to a logger. Otherwise, it will segfault whenever it logs something.

Reply:
openroad creates an db at startup and initializes it with a logger.  We could just use that in the tests and call clear() at the start instead of creating a new one.

Reply:
Alternatively you could make the logger into a static member of the db so create wouldn't need to handle setting the logger